caikit/runtime/servicers/training_management_servicer.py

```diff
@@ -32,16 +38,32 @@ class TrainingManagementServicerImpl:
     """This class contains the implementation of all of the RPCs that are required to run a
     service in Model Mesh as a Model-Runtime."""
 
-    def __init__(self):
-        self.training_manager = TrainingManager.get_instance()
+    # TODO: Align training status enums!
```

Comment:
yeah... do we want to leave two status enums hanging around or just expose the `ModelTrainerBase.TrainingStatus` enum as the external status enum?

Reply:
I'm all for not exposing internal state on external apis but this is just a single enum and I doubt it'll bloat up into way more states, given our previous experience with similar ml training systems

Reply:
Yeah, this is the grossest corner. I think we should probably align on a common enum and use it everywhere. I'll make that change

Reply:
Fixed